base/docker-compose.yml

```diff
@@ -31,3 +31,4 @@ services:
         MOTOR_VERSION: ${MOTOR_VERSION}
         PMAC_VERSION: ${PMAC_VERSION}
         OPCUA_VERSION: ${OPCUA_VERSION}
+        IOCSTATS_VERSION: ${IOCSTATS_VERSION}
```

Comment:
Noticed this now. But move this right below `ETHER_IP` to keep the order from the usage inside scripts. Same thing for ipmiComm (below iocStats).